File: programmers/level1/문자열 다루기 기본/solution2.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(string s) {
    bool answer = true;

    if(s.length()==4||s.length()==6)
    {
       for(char c:s)
        {
            if(!isdigit(c))
            {
                answer=false;
                break;
            }
        }
    }
    else
    {
        answer = false;
    }

    return answer;
}

File: programmers/level2/큰 수 만들기/solution2.js
function solution(number, k) {
    var answer = '';
    var num=number.split("");
    answer = number.substr(k);
    answer = answer.split("");
    
    for(var i=k-1;i>=0;i--)
    {
        for(var j=0;j<number.length-k;)
        {
            var temp = num[i];
            if(num[i]>=answer[j])
            {
                 num[i]=answer[j];
                 answer[j]=temp;
                j++;
            }else{
                break;
            }
        }
    }
    
    answer=answer.join("");
    
    return answer;
}
